lib/geoengineer/resources/aws_sns_topic_subscription.rb

```diff
@@ -39,4 +39,14 @@ def self._fetch_remote_resources
       }
     end
   end
+
+  def self._get_all_subscriptions
+    subs_page = AwsClients.sns.list_subscriptions
```

Comment:
Yea... I tried to implement quickly, and then when I realized that there was `next_marker` and `next_token` and probably others, I gave up.